packages/icons/src/far-away.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
```

Comment:
Don't they need license? 
Please add screenshots of icons in your description in this PR.

Reply:
Added license and screenshots

Reply:
This one is not based on crop_free right?

Reply:
Sorry, fixed